src/groundStation/system.py

```diff
@@ -922,7 +988,17 @@ def __init__(self):
                         'what': 'On, or off',
                         'subPort': 7,
                         'inoutInfo': {
-                            'args': ['>B', '>u2'],  # state, duration (s)
+                            'args': ['<B', '<u2'],  # state, duration (s)
+                            'returns': {
+                                'err': '>b'
+                            }
+                        }
+                    },
+                    'PAUSE_EPS_DEPLOYMENT_ACTION': {
+                        'what': 'Pauses a certain action group deployment for a set time',
+                        'subPort': 8,
+                        'inoutInfo': {
+                            'args': ['>B', '<u4'],  # group, time
```

Comment:
How is the time parameter interpreted? As in unix time or delay time?

Reply:
I'm pretty sure it's a delay. Again, there is nothing that EPS can deploy atm, so this subservice is not tested yet. This is the correct implementation based on two different versions of ICD.

Reply:
Alright. I'm sure you'll find out later